From: Hans de Goede <hdegoede@redhat.com>
Date: Mon, 15 Feb 2016 22:07:50 +0100
Subject: mtt: Fix mtt not being able to tune tv channels

Teletext should use the tv tuner not the radio tuner...

Signed-off-by: Hans de Goede <hdegoede@redhat.com>
---
 x11/vbi-gui.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/x11/vbi-gui.c b/x11/vbi-gui.c
index f4b19da..429c8b2 100644
--- a/x11/vbi-gui.c
+++ b/x11/vbi-gui.c
@@ -917,7 +917,7 @@ static void vbi_station_cb(Widget widget, XtPointer client, XtPointer call)
 	struct v4l2_frequency frequency;
 
 	memset (&frequency, 0, sizeof(frequency));
-	frequency.type = V4L2_TUNER_RADIO;
+	frequency.type = V4L2_TUNER_ANALOG_TV;
 	frequency.frequency = channels[i]->freq;
 	if (-1 == ioctl(vbi->fd, VIDIOC_S_FREQUENCY, &frequency))
 	    perror("ioctl VIDIOCSFREQ");
